configs/mx7ulpea-ucom_lmp_defconfig

```diff
@@ -73,7 +77,9 @@ CONFIG_FSL_LPUART=y
 CONFIG_SPI=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_QSPI=y
+CONFIG_TEE=y
 CONFIG_OPTEE=y
+# CONFIG_OPTEE_TA_AVB is not set
```

Comment:
Same here, is this on by default?

Reply:
seems to be set when I select some other option